src/pages/Validator/ValidatorField.js

```diff
@@ -78,6 +80,22 @@ export default class ValidatorField extends Component {
     this.setState({ html: updatedContent });
   }
 
+  /**
+   * Indicates whether attributes should be shown, and for which entity.
+   * If the name of an entity exists at the start of the token and is followed 
+   * by a dot, returns that entities name. Else, returns empty string.
+   */
+  showAttributes() {
```

Comment:
Possibly rename this to something like `getEntityFromToken`.

Reply:
Done